tests/legacy/integration/deps/setup.py

```diff
@@ -0,0 +1,40 @@
+# Copyright (C) 2015-2023, Wazuh Inc.
+# Created by Wazuh, Inc. <info@wazuh.com>.
+# This program is free software; you can redistribute it and/or modify it under the terms of GPLv2
+import os
+import shutil
+
+from setuptools import setup, find_packages
+
+
+package_data_list = []
+scripts_list = []
+
+
+def get_files_from_directory(directory):
```

Comment:
Its a util that was used only here in the previous qa framework, but not its not used anymore, and I dont think it will be needed, it could be deleted

Reply:
Deleted on commit [`6ca6c89`](https://github.com/wazuh/wazuh/pull/17155/commits/6ca6c8926e10fc54b32963a10dbf9300346cd36d)